tests/cli/test_main_info.py

```diff
@@ -4,86 +4,154 @@
 from __future__ import annotations
 
 import json
-from os.path import isdir
 from typing import TYPE_CHECKING
 
-from conda.base.context import reset_context
+from conda.base.context import context
+from conda.core.envs_manager import list_all_known_prefixes
+from conda.plugins.reporter_backends.console import ConsoleReporterRenderer
 
 if TYPE_CHECKING:
-    from pytest import MonkeyPatch
+    from pytest_mock import MockerFixture
 
     from conda.testing.fixtures import CondaCLIFixture
 
 
-# conda info --root [--json]
-def test_info_root(reset_conda_context: None, conda_cli: CondaCLIFixture):
-    stdout, stderr, err = conda_cli("info", "--root")
-    assert isdir(stdout.strip())
+BASE_KEYS = {"root_prefix"}
+UNSAFE_CHANNELS_KEYS = {"channels"}
+ENVS_KEYS = {"envs"}
+SYSTEM_KEYS = {
+    "sys.version",
+    "sys.prefix",
+    "sys.executable",
+    "conda_location",
+    "env_vars",
+    "site_dirs",
+}
+DETAIL_KEYS = {
+    "platform",
+    "conda_version",
+    "envs_dirs",
+    "pkgs_dirs",
+    "channels",
+    "config_files",
+    "offline",
+    "solver",
+}
+
+
+# conda info --base [--json]
+def test_info_base(conda_cli: CondaCLIFixture) -> None:
+    stdout, stderr, err = conda_cli("info", "--base")
+    assert stdout.strip() == context.root_prefix
     assert not stderr
     assert not err
 
-    stdout, stderr, err = conda_cli("info", "--root", "--json")
+    stdout, stderr, err = conda_cli("info", "--base", "--json")
     parsed = json.loads(stdout.strip())
-    assert isdir(parsed["root_prefix"])
+    assert parsed["root_prefix"] == context.root_prefix
```

Comment:
```suggestion
    assert paths_equal(parsed["root_prefix"] == context.root_prefix
```

Maybe better to check with `conda.common.path.paths_equal()`.